src/js/admin.js

```diff
@@ -1120,7 +1120,7 @@ $(document).ready(function () {
     }
 
     function stateChange(id, state) {
-        id = id ? id.replace(/\s/g, '_') : '';
+        id = id || '';
```

Comment:
Why?

Reply:
because we can?

Reply:
because else states with spaces in its name never get updated on changes ... And space is an allowed character for object ids ... found that while testing controller 2.0 with iqontrol and they have massive ids with spaces inside.

Reply:
In fact the "initial read" for all states and objects read them with their real naes and also show them that way in admin ... then state change comes in and the id gets replaced ... so the "real" state never gets updated with new values